crates/store/re_grpc_client/src/lib.rs

```diff
@@ -176,8 +178,7 @@ async fn stream_recording_async(
         #[cfg(target_arch = "wasm32")]
         let tonic_client = tonic_web_wasm_client::Client::new_with_options(
             redap_endpoint.to_string(),
-            tonic_web_wasm_client::options::FetchOptions::new()
-                .mode(tonic_web_wasm_client::options::Mode::Cors), // I'm not 100% sure this is needed, but it felt right.
```

Comment:
You've tested that this is not needed on web?

Reply:
It definitely isn't, `cors` is the default mode for `fetch`.